lib/restclient/response.rb

```diff
@@ -20,6 +20,10 @@ def body
       String.new(self)
     end
 
+    def json
+      JSON.parse(body) rescue body
```

Comment:
As @ab [pointed out here](https://github.com/rest-client/rest-client/issues/516#issuecomment-255003536) this method should not be "magic". With that `rescue` it is magical in that it sometimes returns a Hash (parsed JSON) and sometimes a Blob String (body).

Also notice that parsing errors are not handled properly. Especially since you're using a gem, it's unpredictable to the users or `RestClient` which kind of exceptions may be raised.